components/automate-cli/cmd/chef-automate/restart.go

```diff
@@ -2,26 +2,76 @@ package main
+	node       string
+	timeout    int
 }
 
-func runRestartServices(cmd *cobra.Command, args []string) error {
+func init() {
+	var restartCmdFlags = RestartCmdFlags{}
+	var restartServicesCmd = &cobra.Command{
+
+		Use:   "restart-services",
+		Short: "restart deployment services",
+		Long:  "Restart services for a deployment",
+		RunE:  runRestartServicesCmd(&restartCmdFlags),
+		Annotations: map[string]string{
+			docs.Tag: docs.BastionHost,
+		},
+	}
+	restartServicesCmd.PersistentFlags().BoolVarP(&restartCmdFlags.automate, "automate", "a", false, "restart chef automate service on automate nodes")
+	restartServicesCmd.PersistentFlags().BoolVar(&restartCmdFlags.automate, "a2", false, "restart chef automate service on automate nodes[DUPLICATE]")
+	restartServicesCmd.PersistentFlags().SetAnnotation("automate", docs.Compatibility, []string{docs.CompatiblewithHA})
+	restartServicesCmd.PersistentFlags().BoolVarP(&restartCmdFlags.chefServer, "chef_server", "c", false, "restart chef automate service on chef-server nodes")
+	restartServicesCmd.PersistentFlags().BoolVar(&restartCmdFlags.chefServer, "cs", false, "restart chef automate service on chef-server nodes[DUPLICATE]")
+	restartServicesCmd.PersistentFlags().SetAnnotation("chef_server", docs.Compatibility, []string{docs.CompatiblewithHA})
+	restartServicesCmd.PersistentFlags().BoolVarP(&restartCmdFlags.opensearch, "opensearch", "o", false, "restart hab-sup service on opensearch nodes")
+	restartServicesCmd.PersistentFlags().BoolVar(&restartCmdFlags.opensearch, "os", false, "restart hab-sup service on opensearch nodes[DUPLICATE]")
+	restartServicesCmd.PersistentFlags().SetAnnotation("opensearch", docs.Compatibility, []string{docs.CompatiblewithHA})
+	restartServicesCmd.PersistentFlags().BoolVarP(&restartCmdFlags.postgresql, "postgresql", "p", false, "restart hab-sup service on postgresql nodes")
+	restartServicesCmd.PersistentFlags().BoolVar(&restartCmdFlags.postgresql, "pg", false, "restart hab-sup service on postgresql nodes[DUPLICATE]")
+	restartServicesCmd.PersistentFlags().SetAnnotation("pg", docs.Compatibility, []string{docs.CompatiblewithHA})
+	restartServicesCmd.PersistentFlags().StringVar(&restartCmdFlags.node, "node", "", "Node Ip address")
+	restartServicesCmd.PersistentFlags().SetAnnotation("node", docs.Compatibility, []string{docs.CompatiblewithHA})
+	restartServicesCmd.PersistentFlags().IntVar(&restartCmdFlags.timeout, "wait-timeout", DEFAULT_TIMEOUT_FOR_RESTART, "This flag sets the operation timeout duration (in seconds) for each individual node during the restart services")
+	RootCmd.AddCommand(restartServicesCmd)
+}
+
+func runRestartServicesCmd(flags *RestartCmdFlags) func(cmd *cobra.Command, args []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		return runRestartServices(cmd, args, flags)
+	}
+}
+
+func runRestartServices(cmd *cobra.Command, args []string, flags *RestartCmdFlags) error {
+	if isA2HARBFileExist() {
+		NodeOpUtils := &NodeUtilsImpl{}
```

Comment:
Why `NodeOpUtils` variable name is in capitals ?

Reply:
changed to nodeOpUtils